.editorconfig

```diff
@@ -157,12 +154,6 @@ csharp_space_between_method_declaration_parameter_list_parentheses = false
 csharp_space_between_parentheses = false
 csharp_space_between_square_brackets = false
 
-# Analyzers
-dotnet_code_quality.CA1052.api_surface = private, internal
-dotnet_code_quality.CA1802.api_surface = private, internal
-dotnet_code_quality.CA1822.api_surface = private, internal
-dotnet_code_quality.CA2208.api_surface = public
```

Comment:
Why are these 5 entries removed?

Reply:
Because they don't belong here.  Some of these rules are disabled, so specifying what surface area we want them applied to isn't useless.  The rest were moved over to the globalconfig next to where we set the severity level.